runtime/src/main/scala/tailcall/runtime/transcoder/Config2Blueprint.scala

```diff
@@ -111,9 +111,10 @@ object Config2Blueprint {
       config.server.baseURL match {
         case Some(baseURL) => TValid.succeed {
             val steps    = field.unsafeSteps.getOrElse(Nil)
-            val host     = baseURL.getHost
-            val port     = if (baseURL.getPort > 0) baseURL.getPort else 80
-            val scheme   = if (baseURL.getProtocol.toLowerCase == "https" || port == 443) Scheme.Https else Scheme.Http
+            val url      = http.baseURL.getOrElse(baseURL)
```

Comment:
Tests required :

- Config -> SDL -> Config should maintain the baseURL set in the HTTP operation.
- baseURL set in the HTTP operation should be used to make HTTP calls when executed.